ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -339,7 +339,7 @@ def test_sum(
     input_dtype, x, axis, castable_dtype = dtype_x_axis_castable
     # ToDo: set as_variable_flags as the parameter generated by test_sum once
     # this issue is marked as completed https://github.com/pytorch/pytorch/issues/75733
-    if "torch" in backend_fw:
+    if "torch" or "jax" in backend_fw:
```

Comment:
> @vedpatwardhan @AnnaTz The paddle version has been changed to `2.5.2`, which might create error in the `paddle` backend. The `jax` backend for the complex datatypes has been resolved. Kindly check!

Yeah it might create an issue with the `paddle` backend but for the `jax` backend we seem to be skipping the gradient tests entirely, but we should only be doing so if we've generated complex dtypes. Thanks @Aaryan562 😄

Reply:
> > @vedpatwardhan @AnnaTz The paddle version has been changed to `2.5.2`, which might create error in the `paddle` backend. The `jax` backend for the complex datatypes has been resolved. Kindly check!
> 
> Yeah it might create an issue with the `paddle` backend but for the `jax` backend we seem to be skipping the gradient tests entirely, but we should only be doing so if we've generated complex dtypes. Thanks @Aaryan562 😄

In my latest commit I have target `jax` backend and its `complex` dtypes.